### PR TITLE
Implemented the Blackboard GUI, now we can watch the states of an sin…

### PR DESCRIPTION
**Pull request by azimme2s**
…gle Product

### CODE REVIEW

File: src/main/java/org/awesomebakery/Start.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public static void main(String[] args) throws Throwable {
 		}
 
 		startBakeryRelatedAgents(scenario, mainContainer);
-		if (cmd.hasOption(OPT_GUI)) {
+		if (!cmd.hasOption(OPT_GUI)) {
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/Start.java]: i dont understand this

File: src/main/java/org/awesomebakery/db/Board.java [hunk 2]
```diff
@@ -94,7 +95,7 @@ private synchronized void addOrder(Order order) {
 			Product product = bakery.getProducts().stream().filter(p -> p.getGuid().equals(entry.getKey())).findFirst()
 					.get();
 			
-			ProductOrder productOrder = new ProductOrder(order, product, entry.getValue());
+			ProductOrder productOrder = new ProductOrder(order, product, entry.getValue());;
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/db/Board.java]: can be removed

File: src/main/java/org/awesomebakery/Start.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (azimme2s) [reply src/main/java/org/awesomebakery/Start.java]: Yeah but the option is currently on false ;)
You can change it with parameters, when you run the Program

[the remark this answers is not in the document]
Comment (mwiktorin) [reply src/main/java/org/awesomebakery/Start.java]: The option depends on your running argument, if you dont set it then no gui, if you set it then gui.

**Pull request merged**